quota/etcd/quotapb/quotapb.proto

```diff
@@ -133,6 +133,10 @@ message UpdateConfigRequest {
   // Fields modified by the update request.
   // For example: "state" or "max_tokens".
   google.protobuf.FieldMask update_mask = 3;
+
+  // If true the updated quota is reset, regardless of the updates's contents.
+  // A reset quota is replenished to its maximum number of tokens.
+  bool reset_quota = 4;
```

Comment:
There are a few reasons why I've added this here:
* I feel this gives us a leaner API
* Update/Reset combos are common enough that it makes sense to bundle them together
* A few config changes may force resets regardless of the param. Eg, the current number of tokens if greater than max_tokens, the new ceiling gets automatically applied. \*

I think there's precedent for this in our APIs, but I can't quite recall where now. At least, I had a quick look at the [API Design Guide](https://cloud.google.com/apis/design/) and couldn't find anything against it.


(\* This is the kind of thing that will go into the README, along with example configs and general usage.)